src/app.py

```diff
@@ -26,4 +27,8 @@
 
 
 if __name__ == '__main__':
-    uvicorn.run(app, host="0.0.0.0", port=3000)
+    uvicorn.run(
+        app,
+        host='0.0.0.0',
+        port=os.environ.get('HTTP_PORT', 300)
```

Comment:
Non conosco la sintassi, in questo caso non mi è chiarissimo, per cosa viene usata questa porta `300`?

Reply:
os.environ.get è un metodo per leggere la env var in python. La porta specificata dovrebbe essere il fallback.
300 è un 3000 che non ci ha creaduto abbastanza :) Ho corretto la porta di default in 3000...

Reply:
ok perfetto, thanks